### DISCUSSION TITLE
Arsonist (Neutral)

### DISCUSSION THREAD

**Discussion by Alf21** [Alf21/TTT2#750] [Role Ideas]
### **Arsonist**
**Description**

The Arsonist is an evil role. His goal is to kill everybody. But he can’t do so by regular means… He needs to burn his victims to feel an ounce of satisfaction.

**Role abilities**

The Arsonist is immune to fire and, by nature, has a 50% bullet resistance. On the flip side: He deals no damage! Then, how does he kill?! 

Easy: He must find people and press “E” on them to douse them in gasoline. This process does not emit sounds or alert the victim in any way. By using his special weapon, the lighter, he can incinerate everybody on the map he has doused before. The incinerated people face certain death.

Doused people are marked on the Arsonist’s scoreboard.

Once the Arsonist has incinerated a certain amount of people and is satisfied, his abilities slowly grow stronger. When he incinerates people, his strength comes back. To deal 100% damage like any other player, he’d have to douse and incinerate at least 1/3 of all players. This means: The earlier an Arsonist is found, the better for everyone else. 

_Optional_
In games with few players, a ConVar should exist to remove the latter mechanic and make the Arsonist always deal no damage, no matter what. This forces him to incinerate everybody with his lighter.
There could also be an option to decide how much ammo the lighter has, i. e. how often the Arsonist can incinerate people.

**Win conditions**

To win, the Arsonist must be the last one standing. 

**Why it might be interesting**

This adds another no-damage role, next to the Jester, only this time around the Arsonist is someone you definitely want to kill. This makes the interactions with both roles a bit deeper.

He’s also unique in a way where he actively seeks the proximity of everybody so he can douse them with E. The Arsonist must find ways to be in close range to everyone without seeming suspicious.

**Interactions with other roles**

When it becomes clear, that the Arso deals no damage, he’ll be quickly prioritized by Traitors, which means that he must find balance between being close to other players but also keeping his abilities secret.
In general, the Arso should stay within crowds for the first part of his game, so he can douse as many people as possible without getting picked off by an evil role.

---

Question: Shall there be an animation for dousing his enemy into gasoline? How long shall the process need? Would be a way too op to stay next to a victim, pressing E and instantly lighter his enemy...

Answer: Hard to say. My thought processes always have my own player group in the back of my head and simply moving close to someone is very suspicious for us already, so I feel you'd need ConVars to balance it yourself, dependent on your playerbase.

Animation: Could either be the Holstered or the Knife wield animation. No new animation work and the latter is universally recognized as being something evil. The former is more neutral and might be better for more alert / small groups of players. If you want, you should also be able to make it work without an animation.
I'd say: ConVar to choose between no animation, holstered and knife animation.

Dousing time: 2-3 seconds default sounds alright but it depends on the amount of players a lot. 5 seconds might sound fair at first but on a server with 20+ people it becomes tedious very quickly. And on the flip side, hugging every player in a 6 man game for 5 seconds with good communication is super suspicious aswell.
ConVar should just be any number in seconds with 0 being instant dousing.

Dousing range: Might also be helpful for more individual balancing. Just have it decide the range where 'E' douses people in units. Default should be about knife range imo (I think it was around 70 units?).

Other ConVars would be lighter ammo (Does the Arso has multiple shots at burning people? Or does he have to douse all of them in one go?) and maybe one that decides whether the Arso deals any regular damage at all.

The main post has the idea that he gets 'buffed' by incinerating people until he can kill normally but I'm not sure if that makes the role too complicated.

Idea by: @Skatcat

**Comment by mexikoedi** [comment]
There is a similar role but not exactly how it is described here: https://steamcommunity.com/sharedfiles/filedetails/?id=2652626080
